portfolio/src/main/webapp/index.html

```diff
@@ -101,14 +101,14 @@ <h2> My Favorite Meme </h2>
     <br/>
 
     <!-- <button onclick="getResponse()" id="helloTxt"> Get Comments </button> -->
-    <form id="commentForm" action="/data-comments" method="POST">
+    <form id="commentForm" onsubmit="postComment(); return false">
```

Comment:
could you just make "postComment()" a boolean and add the "return false" inside of "postComment()"?

Reply:
That sounds smart, will fix

Reply:
It seems like onsubmit="return postComment()" or onsubmit="postComment()" with postComment returning false does not give the intended behavior of not reloading the page since postComment is an async function and instead returns a promise object that must be waited for. 
Will keep the current implementation and merge unless another implementation is preferred to prevent page refreshing on form submit (i.e. changing default form behavior).